From 9e63867ec86485da5bbd02b9856d918f375c882f Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 28 Aug 2012 13:56:16 +0800
Subject: [PATCH 051/281] ENGR00221673 Refine the xPU's device node file
 permission

IPU device
owner: graphics:system, mode: 0660
VPU and ION device
owner: media:system,  mode: 0660

Because all the threads in mediaserver tasks
has the uid:media, gid:system

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 83a37dc..ef30b8a 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -6,19 +6,19 @@
 /dev/video1               0660   system     camera
 /dev/video2               0660   system     camera
 /dev/video16              0660   system     graphics
-/dev/mxc_ipu              0666   system     graphics
+/dev/mxc_ipu              0660   graphics   system
 /dev/mxc_iim              0664   system     media
 /dev/mxs_viim             0664   system     media
 /dev/fsl_cache            0666   system     graphics
 /dev/gsl_kmod             0666   system     graphics
-/dev/mxc_vpu              0666   system     media
+/dev/mxc_vpu              0660   media      system
 /dev/uinput               0666   system     input
 /dev/ttymxc2              0660   system     gps
 /dev/ttya0                0660   system     gps
 /dev/ptya0                0660   system     gps
 /dev/ttya1                0660   system     gps
 /dev/ptya1                0660   system     gps
-/dev/ion                  0666   system     media
+/dev/ion                  0660   media      system
 
 # sysfs properties
 /sys/devices/virtual/input/input*   name        0660  root   input
-- 
1.8.0

